.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+name: ci
+jobs:
+  units:
+    run-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: ${{matrix.java}}
+    - run: java -version
```

Comment:
This yaml file is from https://github.com/googleapis/java-cloud-bom/blob/master/.github/workflows/ci.yaml